component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpAuthenticator.java

```diff
@@ -70,11 +70,12 @@ public static boolean authenticate(HTTPCarbonMessage httpCarbonMessage) {
                             return false;
                         }
                     } catch (IdPClientException e) {
-                        logger.error("Authorization fails.");
+                        logger.error("Authorization process fails for " + usernamePassword[0] + ":" +
```

Comment:
I thin you shouldn't log the password when faling

Reply:
Fixed in 6bf095af9d7a4c00220069badb50517cec553139